### PR TITLE
Create Issue Templates

### LINKED ISSUE #1: Housekeeping - Repository Setup
This issue will be used to track all setup tasks that need to take place before work on user stories can begin.

- [x] Create Develop Branch
- [x] Issue Templates (User Stories)
- [x] Backend Boilerplate
  - [x] ASP.NET Core API Project
  - [x] xUnit Test Project
- [x] Frontend Boilerplate
  - [x] Next.js
  - [x] ESLint + Airbnb Preset
  - [x] Material UI
- [x] CI/CD

### PR DESCRIPTION
**Pull request by miosenpai**
Resolves #1.

### CODE REVIEW

**Review by TianYueDong — APPROVED** [review]
looks good to me

File: .github/ISSUE_TEMPLATE/bug-report-template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Report Template
+about: Use this template to report bugs.
+title: Bug - TITLE
+labels: ''
```
Comment (andy9775) [inline .github/ISSUE_TEMPLATE/bug-report-template.md]: I would add a `bug` label. Makes it easier to search and organize.

File: .github/ISSUE_TEMPLATE/user-story-template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+name: User Story Template
+about: Use this template to create user stories.
+title: User Story - TITLE
+labels: ''
```
Comment (andy9775) [inline .github/ISSUE_TEMPLATE/user-story-template.md]: ditto (`story`?)

**Review by andy9775 — APPROVED** [review]
Just some minor comments, otherwise LGTM!

**Pull request merged**